smartthings-client/README.md

```diff
@@ -117,6 +117,12 @@ devices?.forEach { switchesConfig ->
 }
 ```
 
+### Reference Documentation
+
+Reference documentation can be found here:
+
+https://SmartThingsCommunity.github.io//smartapp-sdk-java/smartthings-client/javadoc/
```

Comment:
Not sure how I feel about the title cased organization in the URL, but it's okay. There is an extra `/` in there, though.

Reply:
This isn't something we chose here...it's because this is in the `SmartThingsCommunity` organization. It's conceivable that it's case-insensitive but I can't test that until it's pushed and wouldn't really want to deviate from the docs even in that small way if possible.

https://help.github.com/en/articles/user-organization-and-project-pages#project-pages-sites

Reply:
URLs aren't case sensitive by definition but this is how GitHub presents it:
![image](https://user-images.githubusercontent.com/1534259/60823974-5ff6e000-a16e-11e9-9acf-386991c7d809.png)